Makefile

```diff
@@ -233,6 +233,14 @@ var: \
 	$(FONTDIR)/var/InterVariable.ttf \
 	$(FONTDIR)/var/InterVariable-Italic.ttf
 
+googlefonts: var
+	gftools fix-family $(FONTDIR)/var/*.ttf \
```

Comment:
One more thing: These commands need to be prefixed with `. $(VENV) ;` else they will fail unless the user first manually modifies their shell's environment.

I.e.

```
gftools fix-family ...
```

becomes

```
. $(VENV) ; gftools fix-family ...
```

Reply:
(Sorry I didn't catch this earlier)